### DISCUSSION TITLE
Some advices about right clicking the flash page.

### DISCUSSION THREAD

**Discussion by Atlas4285** [ruffle-rs/ruffle#3273] [Ideas]
Usually on the past flash version, after right clicking the flash page on the website, it would pop up a list which is similar to other normal page (including back/reload/print/website source code/...). However, when using the Ruffle, it would only have two botton (fullscreen and about Ruffle), which may confusing working a lot. I hope this would can be changed to what it was as before asap.
Really ty for such a useful extension/software!

**Comment by ousia** [comment]
@AtlasZQL,

I’m afraid right-click options emulate the ones the Flash plugin provided.

It will be extended to provide the same functionality that the Flash plugin had.

Or which are the reasons see to ignore that functionality?

**Reply by Atlas4285** [reply]
Actually, I suppose that ruffle can make the flash back/front.
Ummmm, maybe as well as check for the source code or help to download the flash files(.swf).
